docs/index.md

```diff
@@ -0,0 +1,10 @@
+
+# screw·driv·er
+
+> noun
+
+> 1. a tool for turning screws so as to drive them into their place.
+
+> 2. vodka and orange juice served with ice.
+
+> 3. a service for easily building and deploying software.
```

Comment:
you're missing a 4th definition

`4. a software engineer at Yahoo, St.John Johnson`